docs/frequently-asked-questions.rst

```diff
@@ -647,6 +648,20 @@ gas and return your 20 Wei).
 In the above example, the low-level function `call` is used to invoke another
 contract with `p.data` as payload and `p.amount` Wei is sent with that call.
 
+How do I initialize a contract with only a specific amount of wei?
+==================================================================
+
+Currently the approach is a little ugly, but there is little that can be done to improve it.
+You will need to make sure that you have both contracts aware of each other's presence.
```

Comment:
We should. But I'm not sure how to do eloquently. I figured the example would handle that.